build_windows.md

```diff
@@ -16,17 +16,183 @@ Pandoc could be installed from https://pandoc.org/installing.html When performin
+The latest release of the WiX Toolset can be obtained from https://wixtoolset.org/docs/wix3/. Installing it into a clean VM might require
 an additional installation of .NET Framework 3.5 in advance
 ([instructions for adding .NET Framework 3.5 via enabling the Windows feature](https://learn.microsoft.com/en-us/dotnet/framework/install/dotnet-35-windows#enable-the-net-framework-35-in-control-panel))
 
-## Install msys2
+## Building and running without MSYS2
+
+### Install git and go
+
+To build podman you need [git](https://gitforwindows.org/) and [go](https://go.dev). In case you haven't them yet you can 
+install them opening a Windows Terminal and run the following command:
+
+```
+winget install -e GoLang.Go Git.Git
+```
+
+:information_source: For the `PATH` to be reloaded, a restart of your terminal is advised.
+
+### Enable Hyper-V (optional)
+
+Podman on Windows can run on the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/) or 
+on [Hyper-V](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v) . If you want to 
+use Hyber-V and you haven't enabled it yet, you can do it using the following command:
+
+```
+Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All
+```
+
+After running this command a restart of the Windows machine is required.
+
+:information_source: The VM provider used by podman (Hyper-V or WSL) can be configured in the file 
+`%APPDATA%/containers/containers.conf`. More on that later.
+
+### Get the source code
+
+Open a Windows Terminal and run the following command:
+
+```
+git config --global core.autocrlf false
+```
+
+This configures git so that it does **not** automatically convert LF to CRLF. Files are expected to use the Unix LF rather 
+Windows CRLF in the podman git repository.
+
+Then run the command to clone podman git repository:
+
+```
+git clone https://github.com/containers/podman
+```
+
+This will create the folder `podman` in your current directory and clone podman git repository in it.
+
+### Build the podman client for windows
+
+The podman client for Windows can be build with the PowerShell script [winmake.ps1](https://github.com/containers/podman/blob/main/winmake.ps1).
+
+On Windows computers, the ExecutionPolicy is set `Restricted` by default: running scripts is not allowed. You can verify 
+the ExecutionPolicy on your machine with this commmand:
+
+```
+Get-ExecutionPolicy
+```
+
+If the command returns `Restricted` you should change the ExecutionPolicy to `RemoteSigned`:
+
+```
+Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+```
+
+This policy allows the execution of local PowerShell scripts, such as `winmake.ps1` for current user:
+
+```
+# Get in the podman git repository directory
+cd podman
+
+# Build podman.exe
+.\winmake.ps1 podman-remote
+
+# Downlaod gvproxy.exe and win-sshproxy.exe
+# that are needed to execute the podman client
+.\winmake.ps1 win-gvproxy
```

Comment:
do you think we should have an install target in the winmake file that does this and podman  and puts it into the filesystem

Reply:
I don't know. Putting it in the filesystem may be problematic. What if you already have a stable podman installed but just want to test main branch? 

And I was thinking that podman.exe should look for the helpers in it's directory. This was also how I interpretated [this comment](https://github.com/containers/podman/issues/21821#issuecomment-1965901606). This is not working currently and if it is appropriate I can create an issue and work on it.

Another issue is that the version of `gvproxy` and `sshproxy` [is currently hardcoded](https://github.com/l0rd/podman/blob/eb3b15126eaa4f25cd27b8dcb50fb510fdd9ea88/winmake.ps1#L40).

Reply:
if you have a stable podman, fwiw, there will be no impact.  But, what i meant is if you did something like `./winmake install` then it would.